File: openapi/code_samples/python/utilitypayments/accounts_balance/code.py
import requests

url = "https://utilities.reloadly.com/accounts/balance"

payload={}
headers = {
  'Accept': 'application/com.reloadly.utilities-v1+json',
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: openapi/code_samples/nodejs/utilitypayments/accounts_balance/code.js
var request = require('request');
var options = {
  'method': 'GET',
  'url': 'https://utilities.reloadly.com/accounts/balance',
  'headers': {
    'Accept': 'application/com.reloadly.utilities-v1+json',
    'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});
